resources/js/components/modules/data/alcoholModule.js

```diff
@@ -525,8 +525,16 @@ export const alcoholModuleSlides = {
       }
     }
   },
+  // slide  32
+  17:{
+    slide_type: 'emailForm',
+    header:{
+      title:null,
+      text:null
```

Comment:
There's text that appears right after the title and should be either set as text, but the best case is to make it in the JSON as...

>***content***: {
>> ***paragraph***: {
>>> 0: 'Now is a great time to set a wellness goal to practice responsible drinking, or just  improve your overall health!',
>>> 1: 'For example, "Next week I will plan to drink one fewer drink than normal when I go out with friends." ​Or, "I'll work out at least once next week!"'

The template should also be implemented to reflect it as such. You can import **`InfoContent`** and use that to implement the JSON.